ConfirmRedemptionIntent must fail if redeeming less than 1 STWei

failing unit test

```
1) Contract: OpenSTValue ConfirmRedemptionIntent fails to confirm when amountUT does not convert into at least 1 STWei:
     AssertionError: Did not throw as expected
      at Object.module.exports.expectThrow (test/lib/utils.js:79:5)
      at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)
```

--- a/contracts/OpenSTValue.sol
+++ b/contracts/OpenSTValue.sol
@@ -274,6 +274,8 @@ contract OpenSTValue is OpsManaged, Hasher {
     	expirationHeight = block.number + BLOCKS_TO_WAIT_SHORT;
 
     	UtilityToken storage utilityToken = utilityTokens[_uuid];
+    	// minimal precision to unstake 1 STWei
+    	require(_amountUT >= utilityToken.conversionRate);
     	amountST = _amountUT.div(utilityToken.conversionRate);
 
     	require(valueToken.balanceOf(address(utilityToken.simpleStake)) >= amountST);

